src/block_cache/block_io.c

```diff
@@ -440,7 +549,7 @@ __wt_blkcache_write(WT_SESSION_IMPL *session, WT_ITEM *buf, WT_PAGE_BLOCK_META *
          * output requires), it just means the uncompressed version is as good as it gets, and
          * that's what we use.
          */
-        if (compression_failed || buf->size / btree->allocsize <= result_len / btree->allocsize) {
+        if (compression_failed || buf->size < 2 * WT_BLOCK_COMPRESS_SKIP/* || buf->size / btree->allocsize <= result_len / btree->allocsize */) {
```

Comment:
Is the intention to use this commented out code in the future?  If so, I think a TODO comment would be helpful, it's easy to miss.

Reply:
(Same reply as other thread)